src/style/_variables.scss

```diff
@@ -1,38 +1,55 @@
-$gray-base: #000 !default;
-$white: #fff !default;
-$gray-50: #ebebeb !default;
-$gray-100: #d8dde7 !default;
-$gray-200: lighten($gray-base, 62.75%) !default; // #aaa
-$gray-300: lighten($gray-base, 56.5%) !default; // #999
-$gray-400: lighten($gray-base, 50%) !default; // #888
-$gray-500: lighten($gray-base, 43.75%) !default; // #777
-$gray-600: lighten($gray-base, 37.5%) !default; // #666
-$gray-650: lighten($gray-base, 33.3%) !default; // ##555555
-$gray-700: lighten($gray-base, 31.25%) !default; // #555
-$gray-800: lighten($gray-base, 25%) !default; // #444
-$gray-900: lighten($gray-base, 18.75%) !default; // #333
-
-$gray-custom-1: #41444e !default;
-$gray-custom-2: #3b4151 !default;
+// Base Colours
+$black: #000;
+$white: #fff;
+$gray-50: lighten($black, 92%); //ebebeb
+$gray-200: lighten($black, 62.75%); // #a0a0a0
+$gray-300: lighten($black, 56.5%); // #909090
+$gray-400: lighten($black, 50%); // #808080
+$gray-500: lighten($black, 43.75%); // #707070
+$gray-600: lighten($black, 37.5%); // #606060
+$gray-650: lighten($black, 33.3%); // #555555
+$gray-700: lighten($black, 31.25%); // #505050
+$gray-800: lighten($black, 25%); // #404040
+$gray-900: lighten($black, 18.75%); // #303030
+
+$cod-gray: #1b1b1b;
+$bright-gray: #3b4151;
+$mako-gray: #41444e;
+$waterloo-gray: #7d8492;
+$alto-gray: #d9d9d9;
+$mercury-gray: #e4e4e4;
+$concrete-gray: #e8e8e8;
+$alabaster: #f7f7f7;
+$apple-green: #62a03f;
+$green-haze: #009d77;
+$japanese-laurel: #008000;
+$persian-green: #00a0a7;
+$geyser-blue: #d8dde7;
+$dodger-blue: #1391ff;
+$endeavour-blue: #005dae;
+$scampi-purple: #55a;
+$electric-violet: #7300e5;
+$persian-red: #cf3030;
+$mango-tango: #e97500;
+
+// Theme
 
 $color-primary: #89bf04 !default;
 $color-secondary: #9012fe !default;
 $color-info: #4990e2 !default;
 $color-warning: #ff6060 !default;
 $color-danger: #f00 !default;
 
-$_color-post: #49cc90 !default;
-$_color-get: #61affe !default;
-$_color-put: #fca130 !default;
-$_color-delete: #f93e3e !default;
-$_color-head: #9012fe !default;
-$_color-patch: #50e3c2 !default;
-$_color-disabled: #ebebeb !default;
-$_color-options: #0d5aa7 !default;
+$color-primary-hover: lighten($color-primary, .5%) !default;
 
-$color-green: #008000 !default;
-
-$color-primary-hover: #81b10c !default;
+$_color-post: $green-haze !default;
+$_color-get: $doger-blue !default;
```

Comment:
This is causing a build failure now:

```
Module build failed: 
$_color-get: $doger-blue !default;
            ^
      Undefined variable: "$doger-blue".
      in /var/lib/jenkins/workspace/oss-swagger-ui-pr/src/style/_variables.scss (line 46, column 14)
```

Reply:
Whoops, consequence of a rushed changed. Updated 👍